cve_bin_tool/OutputEngine.py

```diff
@@ -38,6 +42,8 @@ def output_cves(self, outfile, output_type=None):
             self.output_json(outfile)
         elif output_type == "csv":
             self.output_csv(outfile)
+        elif output_type == "html":  # for now just generate dummy data
+            self.output_html()
```

Comment:
I think if we just comment out these two lines, the rest of this can be merged without affecting the running of anything.  That'll make it easier for people to take a look at the reports while you keep working.